Makefile

```diff
@@ -25,6 +26,10 @@ grpc:
 verify:
 	pre-commit run --all-files --show-diff-on-failure --color=always
 
+.PHONY: nogpu_tests
+nogpu_tests:
+	pytest -m $(NOGPU_MARKER) $(PYTEST_CI_ARGS)
```

Comment:
Should add `not $(NOGPU_MARKER)` to the unit_tests command, otherwise we run the test twice.

Reply:
Thanks, fixed in a4cb39b.